mmv1/products/vertexai/IndexEndpoint.yaml

```diff
@@ -0,0 +1,99 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: IndexEndpoint
+base_url: projects/{{project}}/locations/{{region}}/indexEndpoints
+create_url: projects/{{project}}/locations/{{region}}/indexEndpoints
+self_link: projects/{{project}}/locations/{{region}}/indexEndpoints/{{name}}
+update_verb: :PATCH
+update_mask: true
+create_verb: :POST
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: https://cloud.google.com/vertex-ai/docs/reference/rest/v1/projects.locations.indexEndpoints/
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+    resource_inside_response: true
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: True
+    allowed:
+      - True
+      - False
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+description: |-
+  An endpoint indexes are deployed into. An index endpoint can have multiple deployed indexes.
+autogen_async: false
+timeouts: !ruby/object:Api::Timeouts
```

Comment:
I think we can remove these, and rely on the 20m default

Reply:
thanks, I made changes along with your suggestion in [8aeccb0](https://github.com/GoogleCloudPlatform/magic-modules/pull/6873/commits/8aeccb0602f3d80d19eede04abfc27a548662cc3).